src/H5webApp.tsx

```diff
@@ -1,40 +1,31 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import { App, getFeedbackMailto, H5GroveProvider } from '@h5web/app';
 import { ServerConnection } from '@jupyterlab/services';
+import { useFirstMountState } from '@react-hookz/web';
 
 const FEEDBACK_EMAIL = 'h5web@esrf.fr';
 
-function TwoRenderApp() {
-  const [isFirstRender, setIsFirstRender] = useState(true);
-
-  useEffect(() => {
-    setIsFirstRender(false);
-  }, []);
-
-  if (isFirstRender) {
-    return null;
-  }
-
-  return (
-    <App
-      getFeedbackURL={(context) => getFeedbackMailto(context, FEEDBACK_EMAIL)}
-      disableDarkMode
-    />
-  );
-}
-
 function H5webApp(props: { filePath: string }) {
   const { filePath } = props;
   const { baseUrl } = ServerConnection.makeSettings();
 
+  const isFirstMount = useFirstMountState();
```

Comment:
I've moved the hook to this component, but hopefully the work around still works. Might be worth adding a comment to explain what the work around is for, because I can't recall exactly (something with the cell not getting the right height?)